android/src/main/java/com/tailscale/ipn/ui/notifier/Notifier.kt

```diff
@@ -56,6 +57,9 @@ object Notifier {
   @OptIn(ExperimentalSerializationApi::class)
   fun start(scope: CoroutineScope) {
     Log.d(TAG, "Starting")
+    if (!::app.isInitialized) {
+      App.get()
+    }
```

Comment:
I think it would be better if this throws and the caller gets a slap on the wrist for creating a notifier without an app.

This is likely my bad for not calling App.start in the shareActivity onStart which is the only way I can see this happening?